lib/pf/Authentication/Source/RADIUSSource.pm

```diff
@@ -172,7 +174,14 @@ sub _fetch_attributes {
 sub check_radius_password {
     my ($self, $radius, $name, $pwd, $nas, @extra) = @_;
 
-    $nas = eval {$radius->{'sock'}->sockhost()} unless defined($nas);
+    if(!defined($nas)) {
+        if($self->nas_ip_address) {
+            $nas = $self->nas_ip_address;
+        }
+        else {
+            $nas = $cluster_enabled ? pf::cluster::management_cluster_ip : $management_network->{Tip};
```

Comment:
Should it be 
```$nas = $cluster_enabled ? pf::cluster::management_cluster_ip :  ($management_network->{Tvip} // $management_network->{Tip});```
Or is the cluster IP is the VIP?

Reply:
There is no more use of 'Tvip' anymore AFAIK, it would be for when you put a `vip` in pf.conf and that isn't being done since many years. We should likely just get that completely removed at some point.

And yes the management_cluster_ip is where the actual VIP for management is now